File: app/src/main/java/org/illegaller/ratabb/hishoot2i/view/CropActivityView.java
package org.illegaller.ratabb.hishoot2i.view;

import android.content.Intent;
import android.graphics.Point;
import android.view.View;
import org.illegaller.ratabb.hishoot2i.view.common.Mvp;
import org.illegaller.ratabb.hishoot2i.view.widget.CropImageView;

public interface CropActivityView extends Mvp.View {
  void setResult(int resultCode, Intent data);

  void setResult(int resultCode);

  void finish();

  void showProgress(boolean isShow);

  String getPathImage();

  Point getPointRatio();

  CropImageView getCropImageView();

  View getViewBtnOk();

  View getViewBtnCancel();
}

File: app/src/main/java/org/illegaller/ratabb/hishoot2i/ui/crop/CropActivityView.kt
package org.illegaller.ratabb.hishoot2i.ui.crop

import android.net.Uri
import org.illegaller.ratabb.hishoot2i.ui.common.Mvp

interface CropActivityView : Mvp.View {
    fun onErrorCrop(throwable: Throwable)
    fun onSuccessCrop(uri: Uri)
}
